e2e/cypress/tests/integration/account_settings/profile/username_spec.ts

```diff
@@ -116,7 +116,7 @@ describe('Settings > Sidebar > General > Edit', () => {
             // # Step 3
             // * Verify that we've logged in as the temp user
             cy.visit(offTopicUrl);
-            cy.uiOpenUserMenu().findByText(`@${newTempUserName}`);
+            cy.uiOpenUserMenu('').findByText(`@${newTempUserName}`);
```

Comment:
Same at https://github.com/mattermost/mattermost-webapp/blob/f5aa82d04ad05d74d60ea359188f2c6afd715d57/e2e/cypress/tests/support/ui/global_header.d.ts#L113 to update the type, like:
```diff
- uiOpenUserMenu(option: string): Chainable;
+ uiOpenUserMenu(item?: string): Chainable;
```
so that this will remain as `cy.uiOpenUserMenu()...`